scrapyd_k8s/launcher/k8s.py

```diff
@@ -122,13 +121,24 @@ def cancel(self, project, job_id, signal):
         return prevstate
 
     def _parse_job(self, job):
-        return {
+        state = self._k8s_job_to_scrapyd_status(job)
+        _job = {
```

Comment:
Can we come up with a better name here? As a casual reader, I would be confused having two jobs here.